include/net/aws_fota.rst

```diff
@@ -3,29 +3,37 @@
 AWS FOTA
 ########
 
-This library combines the :ref:`lib_aws_jobs` and :ref:`lib_fota_download` libraries to create a user-friendly library that can perform firmware-over-the-air (FOTA) update using HTTP and MQTT TLS.
+The Amazon Web Services firmware over-the-air (AWS FOTA) library combines the :ref:`lib_aws_jobs` and :ref:`lib_fota_download` libraries to create a user-friendly library that can perform an over-the-air firmware update using HTTP and MQTT TLS.
 
-It connects to the specified broker using the existing or given certificates and uses a `TLS <https://www.ietf.org/rfc/rfc5246.txt>`_
-connection for the MQTT connection.
+It connects to the specified broker using the existing or given certificates and uses `TLS`_ for the MQTT connection.
 This means that the data sent in each MQTT message is encrypted.
 
-Note that a device connected to the AWS MQTT broker with a valid but different certificate, which has the same security attributes in AWS as the other device, and which is able to subscribe to the same MQTT topic as the other device, will also receive the message sent to that topic.
+Note that other devices that are connected to the same AWS MQTT broker receive the same messages if:
 
-MQTT is used to receive notification that an update is available, and to retrieve metadata about the update.
-HTTP is used to download the update payload.
+* The other device uses valid (but different) certificates that have the same security attributes in AWS as the original device.
```

Comment:
With the "security attributes" - do we mean the policy?

Reply:
Yes. I think you can change it to policy